var/spack/repos/builtin/packages/r-pegas/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RPegas(RPackage):
+    """Functions for reading, writing, plotting, analysing, and
+    manipulating allelic and haplotypic data, including from VCF files,
+    and for the analysis of population nucleotide sequences and
+    micro-satellites including coalescent analyses, linkage
+    disequilibrium, population structure (Fst, Amova) and equilibrium
+    (HWE), haplotype networks, minimum spanning tree and network, and
+    median-joining networks."""
+
+    homepage = "https://cran.r-project.org/web/packages/pegas/index.html"
+    url      = "https://cran.r-project.org/src/contrib/pegas_0.14.tar.gz"
+
+    maintainers = ['dorton21']
+
+    version('0.14', sha256='7df90e6c4a69e8dbed2b3f68b18f1975182475bf6f86d4159256b52fd5332053')
+
+    depends_on('r@3.2.0:+X', type=('build', 'run'))
```

Comment:
This package does not look like it actually depends on X.
```suggestion
    depends_on('r@3.2.0:', type=('build', 'run'))
```

Reply:
The package Cairo requires +X in order to build properly. This is a dep called upon inside of the r package. 
so:
 depends_on('r@3.2.0:+X', ...) 
 setups the following inside of r package:
 depends_on('cairo+X', type(...), when='+X')
 
Without it, the package does not build. Do you have suggestions to fix this otherwise?

Reply:
The `cairo+X` variant is set for the `r` package when the `+X` variant is set for the `r` package. That is not the default though, the default is `r~X`. The reason you were not able to build it without the `+X` setting is because of the dependency on `texlive`. That is not listed as a dependency of `r-pegas` in the DESCRIPTION file and should be removed. See my comments further down in the review about that.